apps/happy-blocks/src/pricing-plans/hooks/pricing-plans.ts

```diff
@@ -0,0 +1,207 @@
+import formatCurrency from '@automattic/format-currency';
+import { useEffect, useState } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import config from '../config';
+import { ApiPricingPlan, PricingPlansConfiguration } from '../types.js';
+
+const DAYS_IN_MONTH = 31;
+const MONTHS_IN_YEAR = 12;
+
+/**
+ * Returns a formatted motnhly price for a plan.
+ */
+const getMonthlyPrice = ( plan: ApiPricingPlan ): string | null => {
```

Comment:
Could we get this from the backend? I don't think this will be 100% correct and perhaps not all the WP.com plans support monthly billing.

Reply:
Huh, interesting.